packages/admin/cms-admin/src/generator/utils/find-root-blocks.ts

```diff
@@ -0,0 +1,61 @@
+import { existsSync } from "fs";
+import { IntrospectionObjectType, IntrospectionQuery } from "graphql";
+
+import { CrudGeneratorConfig } from "../types";
+
+const fallbackLibraryBlocks: { [key: string]: string } = {
+    DamImageBlock: "@comet/cms-admin",
```

Comment:
What about the other blocks in the library? PixelImageBlock, InternalLinkBlock, etc.?

Reply:
yes, more might be added here

- PixelImageBlock won't be used directly, always as child of DamImageBlock (correct me if I'm wrong)
- InternalLinkBlock the same (correct me if i'm wrong)
- LinkBlock is (at least in starter) a common Block